<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaLogin</class>
 <widget class="QDialog" name="VentanaLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>423</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>423</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NaoAplication</string>
  </property>
  <widget class="QLabel" name="lblLogo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/login/logo);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(5, 33, 68);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(7, 34, 64);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Recordar contrase√±a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
background-color: rgb(7, 34, 64);
color: rgb(255, 255, 255);
border-radius: 8px;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbNombre">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(79, 82, 85);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFondo">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>361</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(5, 33, 68);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbPassword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(79, 82, 85);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRecordar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCrear">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-style: solid;
border-width: 1px;
;
border-color: rgb(250, 124, 39);
background-color: rgb(250, 124, 39);
color: rgb(255, 255, 255);
border-radius: 8px;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Crear Usuario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOjoAbierto">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/login/ojoAbierto);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOjoCerrado">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/login/ojoCerrado);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblFondo</zorder>
  <zorder>lblLogo</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>btnAceptar</zorder>
  <zorder>tbNombre</zorder>
  <zorder>label_3</zorder>
  <zorder>tbPassword</zorder>
  <zorder>checkBoxRecordar</zorder>
  <zorder>btnCrear</zorder>
  <zorder>btnOjoAbierto</zorder>
  <zorder>btnOjoCerrado</zorder>
 </widget>
 <resources>
  <include location="imgNaoAplication.qrc"/>
 </resources>
 <connections/>
</ui>
